include/BMS/BMS.hpp

```diff
@@ -150,7 +148,7 @@ class BMS {
      * is high, it represents that the BMS is in a state ready to
      * charge or discharge,
      */
-    EVT::core::IO::GPIO& bmsOK;
+    //    EVT::core::IO::GPIO& bmsOK;
```

Comment:
Commented out code here. Reasoning or delete?

Reply:
I commented out some of the bmsOK stuff instead of deleting it because I think it's going to be reintroduced in the new revision, but I'm not really certain